src/Lunr/Ticks/PHPMailer/PHPMailer.php

```diff
@@ -83,6 +97,178 @@ public function setAnalyticsDetailLevel(AnalyticsDetailLevel $level): void
+
+        $url = $this->Host; // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+
+        // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+        if ($this->Mailer === 'smtp')
+        {
+            $smtpUrl = explode(':', $url);
+
+            if (($smtpUrl[0] === 'smtp' && isset($smtpUrl[2]) && $smtpUrl[2] === '25')
+                || ($smtpUrl[0] === 'smtps' && isset($smtpUrl[2]) && $smtpUrl[2] === '587')
+            )
+            {
+                $url = $smtpUrl[0] . ':' . $smtpUrl[1];
+            }
+
+            unset($smtpUrl);
+        }
+
+        $fields = [
+            'startTimestamp' => $this->startTimestamp,
+            'endTimestamp'   => $endTimestamp,
+            'executionTime'  => (float) bcsub((string) $endTimestamp, (string) $this->startTimestamp, 4),
+            'url'            => $url,
+        ];
+
+        if ($this->analyticsDetailLevel->atleast(AnalyticsDetailLevel::Detailed))
+        {
+            $options = [];
+
+            // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            if ($this->Mailer === 'smtp')
+            {
+                $options = [
+                    'SMTPPort'      => $this->Port, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPHelo'      => $this->Helo, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPSecure'    => $this->SMTPSecure,
+                    'SMTPAutoTLS'   => $this->SMTPAutoTLS,
+                    'SMTPAuth'      => $this->SMTPAuth,
+                    'SMTPUsername'  => $this->Username, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPPassword'  => $this->Password, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPKeepAlive' => $this->SMTPKeepAlive,
+                    'SMTPAuthType'  => $this->AuthType, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPTimeout'   => $this->Timeout, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                ];
+
+                $options += $this->SMTPOptions;
+            }
+
+            $headers = json_encode($this->getMIMEHeaderArray());
+            $options = json_encode($options + $extra);
+
+            $fields['requestHeaders'] = is_bool($headers) ? NULL : $headers;
+            $fields['data']           = $this->prepareLogData($this->MIMEBody);
+            $fields['options']        = is_bool($options) ? NULL : $options;
+
+            unset($headers, $options);
+        }
+
+        $tags = [
+            'type'   => $this->Mailer, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            'status' => $isSent ? '200' : '400',
+            'domain' => parse_url($this->Host, PHP_URL_HOST) ?? $this->Host, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+        ];
+
+        $this->tracingController->stopChildSpan();
```

Comment:
This should be called after `getSpanSpecificTags()`, otherwise the tags might be for a different span. Also `getSpanId()` and `getParentSpanId()` would return IDs for a wrong span.